src/handle_action.cpp

```diff
@@ -471,6 +472,11 @@ static void pldrive( int x, int y, int z = 0 )
     if( !remote && z != 0 && !u.has_trait( trait_PROF_HELI_PILOT ) ) {
```

Comment:
Either you or I will get a merge conflict, we're both touching this line in separate PRs. Just a heads up. 
```suggestion
    if( !remote && z != 0 && ( !u.has_trait( trait_PROF_HELI_PILOT ) || !u.has_trait( trait_PROF_MILI_PILOT ) ) {
```

Reply:
Ah, thanks for the warning. I'm new to all this, will these suggestions avoid the merge conflict?

Reply:
Nope, I don't think we can avoid it.It should be quite easy to fix, but if you don't know what you're doing you can ask me and I can point you to advice.

Reply:
Any advice is appreciated, already feeling a little out of my depth